packages/flutter/test/material/checkbox_list_tile_test.dart

```diff
@@ -1266,6 +1266,49 @@ void main() {
     final Offset offsetPlatform = tester.getTopLeft(platform);
     expect(offsetPlatform, const Offset(736.0, 8.0));
   });
+
+  testWidgets('CheckboxListTile renders with default scale', (WidgetTester tester) async {
+    await tester.pumpWidget(MaterialApp(
+      home: Material(
+        child: CheckboxListTile(
+          value: false,
+          onChanged: (_) {},
+        ),
+      ),
+    ));
+
+    final Transform widget = tester.widget(
+      find.ancestor(
+        of: find.byType(Checkbox),
+        matching: find.byType(Transform),
+      ),
+    );
+
+    expect(widget.transform.getMaxScaleOnAxis(), 1.0);
+  });
+
+  testWidgets('CheckboxListTile respects checkboxScaleFactor', (WidgetTester tester) async {
+    const double scale = 1.5;
+
+    await tester.pumpWidget(MaterialApp(
+      home: Material(
+        child: CheckboxListTile(
+          value: false,
+          onChanged: (_) {},
```

Comment:
Nit: https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#avoid-anonymous-parameter-names